metagpt/document_store/faiss_store.py

```diff
@@ -37,7 +37,11 @@ def _load(self) -> Optional["FaissStore"]:
         return FAISS.load_local(self.raw_data_path.parent, self.embedding, self.fname)
 
     def _write(self, docs, metadatas):
-        store = FAISS.from_texts(docs, self.embedding, metadatas=metadatas)
+        try:
```

Comment:
Use handle_exception decorator if needed. But I still want to ask, is it really necessary to catch anomalies in scenes like this?